packages/core/data-transfer/package.json

```diff
@@ -28,6 +28,8 @@
   "types": "./dist/index.d.ts",
   "scripts": {
     "build": "tsc -p tsconfig.json",
+    "clean": "rm -fr ./dist",
```

Comment:
It would have been nice not to have to declare a clean script & use `tsc --build --clean` on `build:clean`. But it looks like it only removes files related to sourceFiles, (thus it doesn't take deleted files into account).
The advantage would have been that it automatically understands where to look for out files (`tsconfig:outDir`). Here we have to update the scripts if we decide to change the `outDir` path.

Anyway, let's stick with that for now.

Reply:
Agreed, but it looked like this is the only way to have it truly clean. I'll actually switch to rimraf so it's more compatible before merging, since it looks like we already have that dependency in Strapi